src/containers/Projects/ProjectRulesAndGuide/MilestonesAndPayouts/index.tsx

```diff
@@ -0,0 +1,181 @@
+import React, {FC} from 'react';
+import {Icon, IconType} from '@thenewboston/ui';
+
+import AppStoreLogo from 'assets/logos/App Store.png';
+import ChromeLogo from 'assets/logos/Chrome.png';
+import DocsLogo from 'assets/logos/Docs.png';
+import FigmaLogo from 'assets/logos/Figma.png';
+import PlayStoreLogo from 'assets/logos/Play Store.png';
+import {A} from 'components';
+import {useWindowDimensions} from 'hooks';
+
+import {Step1, Step2, Step3, Step4} from '../assets/icons';
+import RoadmapIllustration from '../assets/img/roadmap.png';
+import './MilestonesAndPayouts.scss';
+
+const MilestonesAndPayouts: FC = () => {
+  const SMALL_DEVICE_WIDTH = 768;
```

Comment:
I'm thinking if we should separate the device width constants to a separate constants file, since there is a use case for this in different pages

Reply:
Okay! Let me put these in a separate file.